src/test/java/services/tests/GuestAccountValidateTest.java

```diff
@@ -0,0 +1,44 @@
+package services.tests;
+
+import com.training.base.BaseTest;
+import com.training.services.ga.validate.GAValidationResponse;
+import com.training.utilities.RestEngine;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GuestAccountValidateTest extends BaseTest {
+
+    /**
+     * Positive validation, putting all valid details and checking response
+     */
+    @Test
+    public void testGuestAccountValidation() {
+        Map<String, Object> headerMap = new HashMap();
+        headerMap.put(map.get("AppKeyHeader"), map.get("AppKeyValue"));
+        headerMap.put(map.get("ContentTypeHeader"), map.get("ContentTypeValue"));
+
+        RestEngine restEngine = new RestEngine();
+
+        GAValidationResponse gaValidationResponse =
+                restEngine.getResponsePost(map.get("base_url") + "/validation"
+                        , headerMap
+                        , "{\"email\": \"testPranav@api.com\"}")
+                        .as(GAValidationResponse.class);
```

Comment:
Can we use:
GAValidationResponse gaValidationResponse = new RestEngine().getResponsePost(..)